docs/conf.py

```diff
@@ -71,10 +68,6 @@
     'python': ('https://docs.python.org/3/', None),
 }
 
-# Disable Gitlab as we need to sign in
-linkcheck_ignore = [
-    'https://gitlab.audeering.com',
-]
 
 # HTML --------------------------------------------------------------------
```

Comment:
`'footer_links': False,` is missing in `html_theme_options`

Reply:
and we should set:

```
html_context = {
    'display_github': True,
}
```

Reply:
Good point, added it.

Reply:
OK, added `html_context` as well